Description: Curl's help text changed format
 Fix issue caused by curl 7.74 and later shortening default help
Author: Mat Ellis <excloudymat@gmail.com>
Origin: upstream
Bug-Debian: https://bugs.debian.org/986068
Forwarded: https://github.com/oetiker/SmokePing/issues/266
Applied-Upstream: https://github.com/oetiker/SmokePing/commit/3a290ee5e603782c5ba3bc4ec7294c8971cc88c4

---
Note: this was merged upstream after release of 2.8.2

--- smokeping-2.8.2.orig/lib/Smokeping/probes/Curl.pm
+++ smokeping-2.8.2/lib/Smokeping/probes/Curl.pm
@@ -220,7 +220,7 @@ sub test_usage {
 
 	my $arghashref = $self->features;
 	my %arghash = %$arghashref;
-        my $curl_man = `$bin --help`;
+        my $curl_man = `$bin --help all`;
         
 	for my $feature (keys %arghash) {
 		next if $curl_man =~ /\Q$arghash{$feature}/;
